templates/solutions/financial-services/index.html

```diff
@@ -0,0 +1,487 @@
+          </li>
+          <li>
+            <button class="p-tabs__item"
+                    id="security-and-compliance"
+                    role="tab"
+                    aria-selected="false"
+                    aria-controls="security-and-compliance-tab">Security and compliance</button>
+          </li>
+        </ul>
+        <form class="u-hide--large u-hide--medium">
+          <select name="tabSelect" id="tabSelect">
+            <option value="cloud-confidence-tab" selected="">Cloud confidence</option>
+            <option value="multi-cloud-containers-tab">Multi-cloud containers</option>
+            <option value="ai-tab">AI</option>
+            <option value="security-and-compliance-tab">Security and compliance</option>
+          </select>
+        </form>
+      </div>
+      <div class="col-6 col-medium-3">
+        <div tabindex="0"
+             role="tabpanel"
+             id="cloud-confidence-tab"
+             aria-labelledby="cloud-confidence"
+             aria-hidden="true">
+          <h3 class="p-heading--5">Cloud confidence for the financial sector</h3>
+          <p>
+            Canonical helps you to gain control over your infrastructure. Unlike proprietary solutions, with open source software, there are no license fees, and no vendor lock in. This saves you money now, and in the long run &ndash; with the freedom to switch providers or self-manage.
+          </p>
+          <p>
+            Our infrastructure technologies help further reduce the time and expense of maintaining a data center with advanced lifecycle management features, including built-in automations so you can focus on creating value.
+          </p>
+          <p>
+            Considering migrating to open source? Find out how you can change virtualization software vendors in our <a href="https://ubuntu.com/engage/vmware-to-ubuntu-infrastructure">webinar</a>, or explore which of Canonical's solutions might work for you in our <a href="https://ubuntu.com/engage/vmware-migration-scenarios?_gl=1*1igs0o8*_gcl_au*MTcxMjk5OTEwNy4xNzM5Mjk1MzQ2">whitepaper</a>.
+          </p>
+          <div class="p-cta-block">
+            <a href="/solutions/infrastructure">Run your data center like a cloud&nbsp;&rsaquo;</a>
+          </div>
+        </div>
+        <div tabindex="0"
+             role="tabpanel"
+             id="multi-cloud-containers-tab"
+             aria-labelledby="multi-cloud-containers"
+             aria-hidden="true">
+          <h3 class="p-heading--5">Multi-cloud containers with a trusted dependency tree</h3>
+          <p>
+            Did you know only 41% of organizations are confident in patching vulnerabilities?&sup1; Our Container Build Service provides up to 12 years of guaranteed long term support (LTS) for any open source container image.
+          </p>
+          <p>Choose Canonical, and trust in stable, security maintained containers. Build it once, use it anywhere.</p>
+          <p class="u-text--muted">
```

Comment:
Once we add off screen readers for the footnotes, this should be hidden
```suggestion
          <p class="u-text--muted" aria-hidden="true">
```